views/template-partials/footer.twig

```diff
@@ -26,10 +26,14 @@
               <a href="{{ item.link }}" {{ item.is_target_blank ? 'target="_blank"' }}>{{ item.title }}</a>
             </li>
           {% endfor %}
+          <li>
+              Last update: August 16, 2022
+          </li>
```

Comment:
remove this. we already have an option to add build info.

Reply:
Ok, done.